Gemfile

```diff
@@ -1,36 +1,37 @@
-source 'https://rubygems.org'
+source "https://rubygems.org"
 
-gem 'rails',        '5.0.3'
-gem 'puma',         '3.9.1'
-gem 'sass-rails',   '5.0.6'
-gem 'uglifier',     '3.2.0'
-gem 'coffee-rails', '4.2.2'
-gem 'jquery-rails', '4.3.1'
-gem 'turbolinks',   '5.0.1'
-gem 'jbuilder',     '2.6.1'
+gem "bootstrap-sass", "3.3.7"
+gem "coffee-rails", "4.2.2"
+gem "jbuilder", "2.6.1"
+gem "jquery-rails", "4.3.1"
+gem "puma", "3.9.1"
+gem "rails", "5.0.3"
+gem "sass-rails", "5.0.6"
+gem "turbolinks", "5.0.1"
+gem "uglifier", "3.2.0"
 
 group :development, :test do
-  gem 'sqlite3', '1.3.13'
-  gem 'byebug',  '9.0.6', platform: :mri
+  gem "sqlite3", "1.3.13"
+  gem "byebug", "9.0.6", platform: :mri
+
 end
 
 group :development do
-  gem 'web-console',           '3.5.1'
-  gem 'listen',                '3.0.8'
-  gem 'spring',                '2.0.2'
-  gem 'spring-watcher-listen', '2.0.1'
+  gem "web-console", "3.5.1"
+  gem "listen", "3.0.8"
```

Comment:
alphabetical order
Em nhớ check CI trước khi TO gửi pull nhé: `framgia-ci run --local`